packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx

```diff
@@ -44,6 +44,13 @@ export const WorkingGroupOpening = () => {
   const { showModal } = useModal()
   const { active: activeMembership } = useMyMemberships()
   const { isLoading, opening } = useOpening(id)
+
+  if (opening) {
+    opening.applications = opening.applications?.filter(
```

Comment:
Last thing before merge. Please extract applications to `useMemo`, since filtering array is pretty heavy operation and number of applications hopefully will get pretty big in the future, we don't want do it with every re-render.

`const activeApplication = useMemo(() => .....)`

Reply:
Sorry for not pointing it out at first approach

Reply:
no problem!
I just committed the changes you suggested can you please check now?